jetty9-compat-base/src/main/java/com/google/apphosting/vmruntime/jetty9/VmRuntimeWebAppContext.java

```diff
@@ -185,16 +181,38 @@ protected void doStart() throws Exception {
       setConfigurationClasses(quickstartConfigurationClasses);
     }
 
-    datastoreService = getDatastoreService();
-
     if (quickstartWebXml == null) {
       addEventListener(new ContextListener());
     } else {
       getMetaData()
           .addDescriptorProcessor(preconfigProcessor = new PreconfigureDescriptorProcessor());
     }
 
+    // DatastoreServiceFactory.getDatastoreService();
```

Comment:
The previous code was fetching the datastoreService to "facilitate testing"? but then it is not actually used.
So I think all the code and comments and imports can be removed.